RibonDAO/interface
# Feat/dapp 536

# Description

- Adds an onboarding modal that allows for choosing cause

# Necessary changes
- adds causes context

## Does this pull request close any open issues?

- [dapp-536](https://ribon.atlassian.net/browse/DAPP-536?atlOrigin=eyJpIjoiNjdmNGU3MGQ2ODYyNGQ1OWEzYWVkOGY3YzU0OWUwZGEiLCJwIjoiaiJ9)

## Type of change


## How to test

- Open anonymous tab an check for the third modal on the onboarding. The modal must allow you to choose a cause and filter nonProfits by the chosen cause.
